### ISSUE TITLE
Change the CanLock hook behaviour

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#774]
From WIKI:

## CanLock
`Bool canLock function(self)`

**TODO:** This hook breaks the TARDIS hook principals as it explicitly requires a hook to return true to allow locking.
This should be changed to return false to disallow but otherwise allow by default.

Return true to allow locking.

Need to revisit this

**Comment by AmyJeanes** [comment]
Agreed, in most cases allow by default allows the most flexibility in hooks
